src/02-common-principles.md

```diff
@@ -48,9 +48,15 @@ misunderstanding we clarify them here.
     subject performed multiple conceptually different behaviors (with different
     sets of instructions) they will be considered one (combined) task.
 
-1.  **Event** - a stimulus or subject response recorded during a task. Each
-    event has an onset time and duration. Note that not all tasks will have
-    recorded events (e.g., “resting state”).
+1.  **Event** - something that happens or may be perceived as happening at a particular
+    moment during the recording.
```

Comment:
Should we emphasize that "may be perceived as happening" refers to the perception of a test subject (it **does**, doesn't it??).

```suggestion
1.  **Event** - something that happens or may be perceived by a test subject as happening at a particular
    moment during the recording.
```